spyro/solvers/acoustic_solver_construction_no_pml.py

```diff
@@ -46,6 +48,9 @@ def construct_solver_or_matrix_no_pml(Wave_object):
     Wave_object.solver = fire.LinearSolver(
         A, solver_parameters=Wave_object.solver_parameters
     )
+    # lin_var = fire.LinearVariationalProblem(lhs, rhs + B, u_np1)
```

Comment:
I would remove commented code. They make code harder to read and make it harder to search for word occurrences.

Reply:
Removed